package.json

```diff
@@ -27,7 +27,7 @@
     "linq": "^3.0.8",
     "jsonpath": "^0.2.11",
     "vscode-jsonrpc": "^3.2.0",
-    "autorest-extension-base": "git+https://github.com/olydis/autorest-extension-base.git"
+    "autorest-extension-base": "olydis/autorest-extension-base"
```

Comment:
using a concise format similar to what we use for specifying sway as a dependency on [line 19](https://github.com/Azure/oav/pull/162/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R19)